### DISCUSSION TITLE
Is it possible to batch convert multiple jpeg images to one dicom file?

### DISCUSSION THREAD

**Discussion by pmattioli** [dcm4che/dcm4che#899] [Q&A] [answered]
I need to convert a number of OCT scans belonging to the same series which are stored in separate jpeg files to a single DICOM file. Is this possible, either with this library or any other?

**Comment by gunterze** [comment, marked as answer]
Could be stored as
- [Multi-frame Single Bit Secondary Capture Image](http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_A.8.2.html),
- [Multi-frame Grayscale Byte Secondary Capture Image](http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_A.8.3.html),
- [Multi-frame Grayscale Word Secondary Capture Image](http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_A.8.4.html) or
- [Multi-frame True Color Secondary Capture Image](http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_A.8.5.html)

if all scans have the same dimensions and bit depth, but it is currently **not** supported by dcm4che's [jpg2dcm](https://github.com/dcm4che/dcm4che/blob/master/dcm4che-tool/dcm4che-tool-jpg2dcm/README.md) utility.

**Comment by CHRRA-INSEL** [comment]
> I need to convert a number of OCT scans belonging to the same series which are stored in separate jpeg files to a single DICOM file. Is this possible, either with this library or any other?

I've the same problem with jpg2dcm that does not suppert multiframe. Did you find any solution?
